motion/p5js/green-hyperspace/index.js

```diff
@@ -0,0 +1,28 @@
+var lines = [];
+
+function aline () {
+  this.x = random(0, width);
+  this.y = random(0, height);
+  this.display = function display () {
+    stroke(random(125), random(250), random(125), 255);
+    line(height/2, width/2, this.x, this.y);
+  };
+  this.move = function move() {
+    this.x = this.x + random(-3, 3);
+    this.y = this.y + random(-3, 3);
+  }
+}
+function setup() {
+  createCanvas(500, 500);
```

Comment:
Please add the canvas assignment as stated in the wiki https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code

    var canvas = createCanvas(500, 500); // !this is important
    canvas.parent('sketch'); // !this as well

We need this to display the sketch in the page